### DISCUSSION TITLE
Add an "Available Stock" overview tab in the warehouse GUI

### DISCUSSION THREAD

**Discussion by Kris-Cloutier** [ldtteam/minecolonies#9712] [Ideas]
A huge QoL update would be a tab in the warehouse GUI to see all available stock and be able to remove it directly to our inventory. The couriers just dump items into the warehouse and it is very hard to keep track of where things are.

**Comment by Nik-Was** [comment]
This exists already (though you can't take items directly from it).
It's the chest button in the bottom right of the warehouse menu. When you click it you can see every item in the warehouse, and clicking the question mark button next to an item shows you where exactly it is.
All buildings have that btw, not just the warehouse (but that's where it's the most useful)

**Reply by Kris-Cloutier** [reply]
Yea but still need to go around to each rack to try and find it. Would like to not have to search around to find something.

**Reply by uecasm** [reply]
When you select an item from there, it will draw a red box around the rack that it's in.  You do still have to look for it, but you can see that through walls and it's faster than opening all the racks individually.
